onboardingtaskapp.client/src/components/Product/Create.jsx

```diff
@@ -34,10 +39,25 @@ export default function Create() {
             if (!response.ok) {
                 throw new Error('Network response was not ok');
             }
-            // Handle success
-            window.location.replace("/products");
+            // Handle success           
             console.log('Product created successfully');
```

Comment:
1. Please check my previous comment about console.logs
2. Please implement the API endpoint that I mentioned above